src/type-inference.md

```diff
@@ -72,7 +72,7 @@ inference works, or perhaps this blog post on
 [Unification in the Chalk project]: http://smallcultfollowing.com/babysteps/blog/2017/03/25/unification-in-chalk-part-1/
 
 All told, the inference context stores four kinds of inference variables (as of
-January 2018 <!-- date: 2018-01 -->):
+February 2021 <!-- date: 2021-02 -->):
```

Comment:
This needs to be checked.

Reply:
You had a PR related to this recently, right? I left a comment about `{integer}`? That would be the place to look I think.

Reply:
Yeah here it is: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/enum.InferTy.html. It would be nice to add a link as well.

Reply:
That's the `InferTy` enum, but I wasn't sure if the inference context stores something else.

Reply:
I haven't heard a response on Zulip yet, so should I revert this and leave the old date?

Reply:
Reverted: db75996c0a9a13aa4890c685d77a00ab7947d25f